### ISSUE TITLE
Improve color contrast in search help

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2051]
**What happened**:
light mode:
![image](https://user-images.githubusercontent.com/50813726/112929093-c6135b80-90e5-11eb-87f3-c599ad86d3d2.png)
The contrast ratio is 2.67, which doesn't pass AA rating.

dark mode:
![image](https://user-images.githubusercontent.com/50813726/112929298-20acb780-90e6-11eb-8a2d-be474b80fab1.png)
The contrast ratio is 5.24, pass AA but we can make it AAA.
**What should have happened**:

**How to reproduce it (as precise as possible)**:

**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:
